src/MIDebugEngine/Engine.Impl/DebuggedThread.cs

```diff
@@ -226,6 +226,18 @@ private ThreadContext CreateContext(TupleValue frame)
             ulong? pc = frame.TryFindAddr("addr");
             MITextPosition textPosition = MITextPosition.TryParse(frame);
             string func = frame.TryFindString("func");
+
+            if (this._debugger.MICommandFactory.Mode == MIMode.Lldb)
+            {
+                if (func.Contains("("))
+                {
+                    // LLDB adds the types of variables to the func. This doesn't conform 
+                    // the mi protocol and results in duplicate data when the ad7 stackframe
+                    // object does the same thing. 
+                    func = func.Substring(0, func.IndexOf('('));
```

Comment:
There is certainly a risk of that but AFAIK, it shouldn't be. I don't believe '(' is a valid identifier. Do we have any cases where decorated names are shown in lldb? I haven't seen any. Arguably, the correct fix is to remove this from lldb-mi since it doesn't conform to the protocol.